jedi/_compatibility.py

```diff
@@ -34,8 +34,32 @@ def close(self):
         del self.loader
 
 
-def find_module_py33(string, path=None):
-    loader = importlib.machinery.PathFinder.find_module(string, path)
+def find_module_py34(string, path=None):
+    implicit_namespace_pkg = False
+    spec = None
+    loader = None
+
+    if isinstance(path, ImplicitNSInfo):
+        old_dotted_path = path.name
+        path = path.paths
+    spec = importlib.machinery.PathFinder.find_spec(string, path)
+    if hasattr(spec, 'origin'):
+        origin = spec.origin
+        implicit_namespace_pkg = origin == 'namespace'
```

Comment:
Here I am just assigning `implicit_namespace_pkg` to the result of the  evaluated expression `origin == 'namespace'`
 It is the same thing as saying:
`implicit_namespace_pkg  = True if origin == 'namespace' else False
` 
I'll refactor this to make it clearer and I'll change `implicit_namespace_pkg` to `is_implicit_namespace_pkg` to make it more explicit that it is a boolean value.